railties/lib/rails/application/configuration.rb

```diff
@@ -489,6 +494,10 @@ def default_log_file
         f
       end
 
+      def default_log_file_size
+        100 * 1024 * 1024
+      end
```

Comment:
This method might not be worth it. I'd move this directly on the `self.log_file_size = ` line.

Reply:
yeah, less is merrier